test/value-contentRelationship.test.ts

```diff
@@ -32,6 +33,7 @@ test("supports custom model", (t) => {
 
 	// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 	t.true(customModel.config?.tags!.every((tag) => actual.tags.includes(tag)));
+	t.is(actual.text, "Aliquam etiam");
```

Comment:
⚠️ #issue: We should avoid hard-coding generated values in tests since they might change. In this case, we could check that `actual.text` is any string.

The same applies for the other tests where the value of `text` is checked against a hard-coded generated value.

Reply:
Thanks, good to know. Fixed ✅